### ISSUE TITLE
"Limit FPS" option is tied to screen refresh rate

### ISSUE THREAD

**Issue by mbc07** [mitxela/swotGB#1]
It works correctly for anyone using a 60 Hz screen refresh rate, but the emulator will run too fast on higher refresh rate screens, even with the "Limit FPS" option enabled...

**Comment by mitxela** [comment]
Yeah I can believe that, didn't bother limiting it properly. The real target is meant to be 59.7FPS anyway. I'll fix it at some point.

**Comment by mitxela** [comment]
Hi, take a look at the latest version. I don't actually have a way of testing it since I don't have a 144Hz monitor but in theory it should work now.

**Comment by mbc07** [comment]
Just tested the most recent version on my 75 Hz monitor and indeed the issue seems fixed. The built-in FPS counter keeps oscillating between 57 and 62 FPS very very quickly, so, while this might not be a fixed 60 FPS lock it still is way better than before, where the limit would be the screen refresh rate...
